elide-core/src/main/java/com/yahoo/elide/core/exceptions/HttpStatusException.java

```diff
@@ -19,7 +19,7 @@
  * Creates fast exceptions without stack traces since filter checks can throw many of these.
  */
 @Slf4j
-public class HttpStatusException extends RuntimeException {
+abstract public class HttpStatusException extends RuntimeException {
```

Comment:
*sigh* this is _very_ pedantic (sorry about that) but it's more common to see `public abstract class` than `abstract public class` (i.e. flip `abstract` and `public`).

Would you mind updating this so we can remain consistent?